### ISSUE TITLE
Where is readme.md?

### ISSUE THREAD

**Issue by PKURio** [Ricardo-Evans/lucis-compiler-java#1]
I am a beginner at compiling. Your code is elegant, but a little hard to follow. So can you show your readme and I can learn more about your codes. Thx, buddy.

**Comment by Ricardo-Evans** [comment]
Currently the main readme is not focused due to large uncertainty of the future work, but you can find a readme about the compiler framework in directory framework.

**Comment by Ricardo-Evans** [comment]
And of course you can write a readme for the project, which is in urge need.
